src/agent/multitype_queue/src/persistence_factory.cpp

```diff
@@ -3,10 +3,10 @@
 
 #include <stdexcept>
```

Comment:
Since we removed the `string` header from the hpp file, we need to add it here in the source file.